README.md

```diff
@@ -24,13 +24,40 @@ Data::Validate::Sanctions - Validate a name against sanctions lists
 
 Data::Validate::Sanctions is a simple validitor to validate a name against sanctions lists.
 
-The list is from [https://www.treasury.gov/ofac/downloads/sdn.csv](https://www.treasury.gov/ofac/downloads/sdn.csv), [https://www.treasury.gov/ofac/downloads/consolidated/cons\_prim.csv](https://www.treasury.gov/ofac/downloads/consolidated/cons_prim.csv)
+The list is from [https://www.treasury.gov/ofac/downloads/sdn_xml.zip](https://www.treasury.gov/ofac/downloads/sdn_xml.zip), [https://www.treasury.gov/ofac/downloads/consolidated/consolidated.xml](https://www.treasury.gov/ofac/downloads/consolidated/consolidated.xml), and [http://hmt-sanctions.s3.amazonaws.com/sanctionsconlist.csv](http://hmt-sanctions.s3.amazonaws.com/sanctionsconlist.csv)
```

Comment:
I think the extra labels/names on the links are unnecessary, just place directly like so

```suggestion
The list is from the following sources:

- https://www.treasury.gov/ofac/downloads/sdn_xml.zip
- https://www.treasury.gov/ofac/downloads/consolidated/consolidated.xml
- http://hmt-sanctions.s3.amazonaws.com/sanctionsconlist.csv
```